obs/preinstallimage-bios.sh.in

```diff
@@ -725,15 +725,25 @@ BindsTo=fty-license-accepted.service
 After=fty-license-accepted.service
 EOF
 
+# This should only work with recent systemd and Debian 10
+# See the fix below
 mkdir -p /lib/systemd/system/nut.target.d
 cat > /lib/systemd/system/nut.target.d/fty.conf  << EOF
 # Remove nut.target from multi-user.target, to avoid breaking
 # the fty-license-accepted Requirement on multi-user.target.
 # To achieve this, simply set WantedBy to empty
 [Install]
 WantedBy=
+WantedBy=bios.target
 EOF
 
+# For systemd 215 at least, dropin for [Install] are not satisfied
+# So hotfix nut.target directly
+# First, disable the target to remove the .wants symlink
+/bin/systemctl disable nut.target
+sed -i "s/WantedBy=multi-user.target/WantedBy=bios.target/" /lib/systemd/system/nut.target
```

Comment:
LGTM... stylistically, I prefer 

`sed -e ... -e ... -i filename`

as more readable what we in-place-edit